components/skin/ui/src/main/resources/PhenoTips/Skin.xml

```diff
@@ -794,10 +795,14 @@ h1:target, h2:target, h3:target {
   float: right;
 }
 .language-block .rightmenu a {
-  float: right;
+  color: $theme.menuLinkColor;
   cursor: default !important;
-  text-decoration: blink;
-  padding-top: 0;
+  float: right;
+  font: normal normal 700 0.8em/1.8em sans-serif;
```

Comment:
If we get rid of uppercase (as per my comment on line `804`), this line should be  `font: normal normal 400 1em/1.4em sans-serif` to better blend into the rest of the header.